tests/fixtures/sample-with-diagram.adoc

```diff
@@ -52,6 +52,21 @@ digraph g {
 }
 ----
 
+== ASCIIToSVG
```

Comment:
Blocking: since you're adding this sample, could you add a corresponding test like other diagram systems?

Reply:
I'm not quite sure what you are asking for with this comment.   With the change made to `sample-with-diagram.adoc`, a2s is being exercised by `@test "We can generate an HTML document with a diagram with asciidoctor-diagram as backend"`.  I verified that the new test data causes a [fail](https://github.com/asciidoctor/docker-asciidoctor/pull/372#issuecomment-1613446738) against the current image.

As far as I can see, that testing approach is  consistent with the tests for the other diagram systems.  Could you be more specific?

Reply:
My apologies: I did not check the comment https://github.com/asciidoctor/docker-asciidoctor/pull/372#issuecomment-1613446738 carefully.

The fact you're updating this sample implies the "nominal" generation case is proving that a2s is used and you even demonstrated it in the command 👍